packages/next-slicezone/package.json

```diff
@@ -10,7 +10,7 @@
     "build": "rollup -c",
     "prepublishOnly": "yarn build && yarn test",
     "lint": "eslint --ext .js,.jsx .",
-    "security-audit:prod": "yarn audit --production --only=prod --audit-level=low",
+    "security-audit:prod": "yarn audit --groups dependencies --audit-level=high",
```

Comment:
I don't think this is the way to solve this problem I think. This line mean that we will ignore moderate security issues.